### ISSUE TITLE
CRAN Distribution probably not possible with our dlls

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/OSPSuite-R#75]
It seems that cran needs to be able to compile the dll (c++ or C)
In our case, it won't work as far as I can tell

@PavelBal Can you do some research here. How can we embed some .NET dll in an R package that should be in CRAN?

Alternatively, we do not bother with CRAN at all /cc @Yuri05 

Still where do we put those dlls?

**Comment by PavelBal** [comment]
The general policy is indeed that everything should be submitted as source and must be automatically compiled and work on Windows/Linux/MacOS. It appears, however, that there is at least a possibility to "negotiate" with the maintainers of CRAN about the issue.

I would propose to contact the CRAN submission team and describe our situation. Maybe they will allow to distribute the compiled DLL. This will be definitely the fastest way to find out...

To summarize, what DLLs will be supplied with the package? Can we provide the source for _any_ of them?

**Comment by msevestre** [comment]
Even more of a Problem:
Our package won't work on mac and we may need two different packages (windows and Linux because we have different native assemblies!)

As far as I am concerned, cran availability is very low on list now.

**Comment by PavelBal** [comment]
Yes, let's postpone this...

**Comment by msevestre** [comment]
closing for now
